journal.py

```diff
@@ -8,13 +8,20 @@
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import scoped_session, sessionmaker
 from zope.sqlalchemy import ZopeTransactionExtension
+import transaction
 import datetime
-from pyramid.httpexceptions import HTTPFound
+from pyramid.httpexceptions import (HTTPFound, HTTPForbidden,
+                                    HTTPMethodNotAllowed, HTTPNotFound)
 from sqlalchemy.exc import DBAPIError
+from sqlalchemy.orm.exc import NoResultFound
 from pyramid.authentication import AuthTktAuthenticationPolicy
 from pyramid.authorization import ACLAuthorizationPolicy
 from pyramid.security import remember, forget
 from cryptacular.bcrypt import BCRYPTPasswordManager
+from markdown2 import Markdown
```

Comment:
Interesting.  I haven't used this one before.  I'd love your feedback on how it compares to the other `markdown` library.